docs/providers/Analytics.tsx

```diff
@@ -48,6 +67,14 @@ export function reportPageView(url: string) {
   });
 }
 
+export function reportPageViewForGA4(url: string) {
+  window?.gtag?.('config', GA4_MEASUREMENT_ID, {
```

Comment:
```suggestion
  window?.ga4tag?.('config', GA4_MEASUREMENT_ID, {
```

Follow naming change above.